src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,205 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+
+    static class Node<T> {
+        private T value;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(Node<T> prev, T value, Node<T> next) {
```

Comment:
```suggestion
        Node(Node<T> prev, T value, Node<T> next) {
```
```suggestion
        private Node(Node<T> prev, T value, Node<T> next) {
```
won't be accessible outside of the file either way